pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdPersistentTopics.java

```diff
@@ -93,420 +93,420 @@ private Topics getPersistentTopics() {
         return persistentTopics;
     }
 
-    @Parameters(commandDescription = "Get the list of topics under a namespace.")
+    @Command(description = "Get the list of topics under a namespace.")
     private class ListCmd extends CliCommand {
-        @Parameter(description = "tenant/namespace", required = true)
-        private java.util.List<String> params;
+        @Parameters(description = "tenant/namespace", arity = "1")
+        private String namespaceName;
 
         @Override
         void run() throws PulsarAdminException {
-            String namespace = validateNamespace(params);
+            String namespace = validateNamespace(namespaceName);
             print(getPersistentTopics().getList(namespace));
         }
     }
 
-    @Parameters(commandDescription = "Get the list of partitioned topics under a namespace.")
+    @Command(description = "Get the list of partitioned topics under a namespace.")
     private class PartitionedTopicListCmd extends CliCommand {
-        @Parameter(description = "tenant/namespace", required = true)
-        private java.util.List<String> params;
+        @Parameters(description = "tenant/namespace", arity = "1")
+        private String namespaceName;
 
         @Override
         void run() throws PulsarAdminException {
-            String namespace = validateNamespace(params);
+            String namespace = validateNamespace(namespaceName);
             print(getPersistentTopics().getPartitionedTopicList(namespace));
         }
     }
 
-    @Parameters(commandDescription = "Grant a new permission to a client role on a single topic.")
+    @Command(description = "Grant a new permission to a client role on a single topic.")
     private class GrantPermissions extends CliCommand {
-        @Parameter(description = "persistent://tenant/namespace/topic", required = true)
-        private java.util.List<String> params;
+        @Parameters(description = "persistent://tenant/namespace/topic", arity = "1")
```

Comment:
Is `Parameter#arity` can fully replace CliCommand#getOneArgument ?

Reply:
Yes.